consensus/polybft/blockchain_wrapper.go

```diff
@@ -196,5 +201,5 @@ func (s *stateProvider) Call(addr ethgo.Address, input []byte, opts *contract.Ca
 // Txn is part of the contract.Provider interface to make Ethereum transactions. We disable this function
 // since the system state does not make any transaction
 func (s *stateProvider) Txn(ethgo.Address, ethgo.Key, []byte) (contract.Txn, error) {
```

Comment:
I would leave panic here, this is just to notify developer that this object should not be used for something for which is not meant

Reply:
I agree with @igorcrevar. This can never happen.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1077/commits/9f8f1c2022511ec0bc37851484a86adc3bd311ca